master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -163,7 +163,16 @@ private[celeborn] class Master(
   }
   masterSource.addGauge(MasterSource.WORKER_COUNT) { () => statusSystem.workers.size }
   masterSource.addGauge(MasterSource.LOST_WORKER_COUNT) { () => statusSystem.lostWorkers.size }
+  masterSource.addGauge(MasterSource.APPLICATION_COUNT) { () =>
+    statusSystem.appHeartbeatTime.size
+  }
   masterSource.addGauge(MasterSource.PARTITION_SIZE) { () => statusSystem.estimatedPartitionSize }
+  masterSource.addGauge(MasterSource.PARTITION_WRITTEN) { () =>
+    statusSystem.partitionTotalWritten.sum()
```

Comment:
This variable partitionTotalWritten will be set to 0 periodically. So you can use this variable directly.